docs/api/configuration.md

```diff
@@ -0,0 +1,64 @@
+# Configuration
+
+Description: TODO.
+
+### Options
+
+- `appium` (`Object?`):
+  - `baseUrl` (`String?`): Base URL of Appium server. Defaults to "http://localhost:4723/wd/hub".
+  - `username` (`String?`): Username for server authentication - useful when connecting to SauceLabs and BrowserStack.
+  - `password` (`String?`): Password for server authentication - useful when connecting to SauceLabs and BrowserStack.
+- `ui` (`String?`): Interface to use when writing tests (e.g. `describe` and `it`). Defaults to "bdd". 
+- `specs` (`Array<String>`): List of spec file paths.
+- `reporters` (`Array<Function>?`): List of reporters to provide insight throughout your test run. Defaults to the [Basic](#basic) reporter. 
+- `plugins` (`Array<Function>?`): List of plugins to provide functionality throughout your test run.
+- `maxDevices` (`Number?`): Maximum number of `devices` to run in parallel. Defaults to 1 (series).
+- `devices` (`Array<Object>`):
+  - `specs` (`Array<String>?`): List of spec file paths. This will be in addition to the top-level `specs`.
+  - `waitForInterval` (`Number?`): Interval duration in milliseconds to poll the expectation when using waits within tests. This will override the top-level `waitForInterval`.
+  - `waitForTimeout` (`Number?`): Maximum duration in milliseconds before timing out when using waits within tests. This will override the top-level `waitForTimeout`.
+  - `testTimeout` (`Number?`): Maximum duration in milliseconds to wait for a test to complete. This will override the top-level `maxTestTimeout`.
+  - `maxRetries` (`Number?`): Maximum number of retries for failing tests. This will override the top-level `testTimeout`.
```

Comment:
Use `maxTestRetries`. This would make it easier to have the concept of `maxSpecRetries` to retry failing spec files.

Reply:
Override sentence contains typo.